packages/keycloak-user-management/package.json

```diff
@@ -56,8 +60,7 @@
     "formik-antd": "^2.0.1",
     "lodash": "^4.17.20",
     "uuid": "^8.3.1",
-    "yup": "^0.29.3",
-    "i18next": "^19.8.4"
```

Comment:
is `i18next` removal here intentional, is it not being used in this package?

Reply:
@p-netm https://github.com/OpenSRP/web/pull/412/files#diff-67f0320007f21c84b55e48ce5f26289b41794fcd484122c248861414f37b9ab8R43